src/main.ts

```diff
@@ -56,12 +58,25 @@ export function run(
   downloadUrl: string = DOWNLOAD_URL,
   executable: string = EXECUTABLE,
   coverageCommand: string = DEFAULT_COVERAGE_COMMAND,
+  workingDirectory: string = DEFAULT_WORKING_DIRECTORY,
   codeClimateDebug: string = DEFAULT_CODECLIMATE_DEBUG,
   coverageLocationsParam: string = DEFAULT_COVERAGE_LOCATIONS,
   coveragePrefix?: string
 ): Promise<void> {
   return new Promise(async (resolve, reject) => {
     let lastExitCode = 1;
+    if (workingDirectory) {
+      debug(`Changing working directory to: ${workingDirectory}`);
+      try {
+        await chdir(workingDirectory);
```

Comment:
```suggestion
chdir(workingDirectory);
```

You don't need to `await` this, the [`chdir()`](https://nodejs.org/api/process.html#process_process_chdir_directory) method doesn't return a promise.